pyls/plugins/jedi_completion.py

```diff
@@ -20,26 +20,25 @@ def pyls_completions(document, position):
 
 def _label(definition):
     if definition.type in ('function', 'method'):
-        params = ", ".join(param.name for param in definition.params)
-        return "{}({})".format(definition.name, params)
+        params = ', '.join(param.name for param in definition.params)
+        return '{}({})'.format(definition.name, params)
 
     return definition.name
 
 
 def _detail(definition):
-    return "builtin" if definition.in_builtin_module() else definition.parent().full_name or ""
+    return 'builtin' if definition.name.startswith('_') else definition.parent().full_name or ''
```

Comment:
I'd remove this entirely, not sure it's valid to say "__dict__" is a builtin, instead of being an attribute on the module.

`return definition.parent().full_name or ''`

Reply:
isn't dict a built in?